pkgs/by-name/pr/protonmail-bridge-gui/package.nix

```diff
@@ -0,0 +1,105 @@
+{ lib
+, stdenv
+, pkg-config
+, libsecret
+, cmake
+, ninja
+, qt6
+, grpc
+, protobuf
+, zlib
+, gtest
+, sentry-native
+, protonmail-bridge
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "protonmail-bridge-gui";
+
+  inherit (protonmail-bridge) version src;
+
+  patches = [
+    ./use-nix-googletest.patch
+    ./designator-order-fix.patch
+  ];
+
+  cmakeFlags = [
+    "-DCMAKE_BUILD_TYPE=Release"
+    "-DBRIDGE_APP_FULL_NAME=ProtonMailBridge" # Should be 'Proton Mail Bridge' but https://github.com/NixOS/nixpkgs/issues/114044
+    "-DBRIDGE_VENDOR=ProtonAG" # Should also be 'Proton AG'
+    "-DBRIDGE_REVISION=${finalAttrs.src.rev}"
+    "-DBRIDGE_TAG=v${finalAttrs.version}"
+    "-DBRIDGE_BUILD_ENV=dev"
+    "-DBRIDGE_APP_VERSION=${finalAttrs.version}"
+  ];
+
+  nativeBuildInputs = [
+    pkg-config
+    cmake
+    ninja
+    qt6.qtbase
+    qt6.qtdeclarative
+    qt6.qtwayland
+    qt6.qtsvg
+    qt6.wrapQtAppsHook
+  ];
+
+  buildInputs = [
+    libsecret
+    grpc
+    protobuf
+    zlib
+    gtest
+    sentry-native
+  ];
+
+  sourceRoot = "source/internal/frontend/bridge-gui";
+
+  postPatch = ''
+    # Bypass `vcpkg` by deleting lines that `include` BridgeSetup.cmake
+    find . -type f -name "CMakeLists.txt" -exec sed -i "/BridgeSetup\\.cmake/d" {} \;
+
+    # Modify DeployLinux.cmake to use the available ICU version
+    sed -i 's/libicu\(i18n\|uc\|data\)\.so\.56/libicu\1.so/g' bridge-gui/DeployLinux.cmake
+  '';
+
+  installPhase = ''
+    runHook preInstall
+
+    # Install the GUI binary
+    install -Dm755 bridge-gui/bridge-gui $out/lib/bridge-gui
+
+    # Symlink the backend binary from the protonmail-bridge (CLI) package
+    ln -s ${protonmail-bridge}/bin/protonmail-bridge $out/lib/bridge
+
+    # Symlink the GUI binary
+    mkdir -p $out/bin
+    ln -s $out/lib/bridge-gui $out/bin/protonmail-bridge
+
+    # Install assets
+    install -Dm644 /build/source/dist/proton-bridge.desktop -t $out/share/applications
+    install -Dm644 /build/source/dist/bridge.svg $out/share/icons/hicolor/scalable/apps/protonmail-bridge.svg
+
+    runHook postInstall
+  '';
+
+  meta = with lib; {
```

Comment:
Tip:

1. Remove `with lib;`
2. Sort alphabetically

(just like the previous PR)